opfront/store.py

```diff
@@ -0,0 +1,12 @@
+from opfront.model import Model
+from opfront.resource import OpfrontResource
+
+
+class StoreResource(OpfrontResource):
+
+    def _make_model(self, **args):
+        model = super()._make_model(**args)
+        if getattr(model, 'banner', None):
```

Comment:
Being super picky with the zen of python would say you validated the 'explicit is better then implicit'. You could add ```is not None``` to you if statement